views/_footer.html

```diff
@@ -1,7 +1,12 @@
 {% block footer %}
-<footer class="site-footer">
-  <div class="footer-wrapper">
-    <p>Footer Text Goes Here</p>
+<footer class="footer">
+  <div class="footer--content">
+    <p>
```

Comment:
Removed footer completely because of irl discussion about whether or not we actually needed one. (  we don't )